tests/app/views/contexts/test_list_collector.py

```diff
@@ -1,11 +1,11 @@
-from app.data_model.answer import Answer
+import pytest
 from app.utilities.schema import load_schema_from_name
 from app.views.contexts import list_collector
 from app.questionnaire.questionnaire_schema import DEFAULT_LANGUAGE_CODE
 
-
+@pytest.mark.usefixtures('app')
```

Comment:
I've been getting lint errors for these fixtures. Not sure what has changed, but this seems to resolve it,.